tests/PhoneValidatorTest.php

```diff
@@ -345,10 +346,11 @@ public function testValidatePhoneLenient()
         ]));
     }
 
+    /**
+     * @expectedException \Propaganistas\LaravelPhone\Exceptions\InvalidParameterException
+     */
     public function testValidatePhoneFaultyParameters()
     {
-        $this->setExpectedException('Propaganistas\LaravelPhone\Exceptions\InvalidParameterException');
```

Comment:
Oops sorry, bold typo, I actually meant the `expectException()` method which is present since 5.2.